src/graphql/auth.rs

```diff
@@ -78,6 +78,22 @@ impl<'r> AdminRequest<'r> {
     }
 }
 
+#[derive(Debug, Serialize)]
+#[cfg_attr(test, derive(Deserialize))]
```

Comment:
I think I left the `cfg_attr` on the other request by mistake. Pretty sure you don't need it

Reply:
So do they both need to go, or just the one you highlighted?

Reply:
I think they can all go. Delete them all and see if the tests still pass?

Reply:
Tests pass without them so they're gone.